src/GovUk.Education.ExploreEducationStatistics.Common/Services/PublicBlobStorageService.cs

```diff
@@ -0,0 +1,23 @@
+#nullable enable
+using Azure.Storage.Blobs;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Services
+{
+    public class PublicBlobStorageService : BlobStorageService
```

Comment:
I would go:

```
public class PublicBlobStorageService : BlobStorageService, IPublicBlobStorageService
```

here, and not have BlobStorageService implement IPrivateBlobStorageService and IPublicBlobStorageService anymore if I were you?